website/docs/r/kms_ciphertext.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "aws"
+page_title: "AWS: aws_kms_ciphertext"
+sidebar_current: "docs-aws-resource-kms-ciphertext"
+description: |-
+    Provides ciphertext encrypted using a KMS key
+---
+
+# Resource: aws_kms_ciphertext
+
+The KMS ciphertext resource allows you to encrypt plaintext into ciphertext
```

Comment:
Nit: We should include a cross referencing link to the `aws_kms_ciphertext` data source and explain the difference between the two.